complex/fabtest.h

```diff
@@ -88,6 +97,9 @@ struct ft_xcontrol {
 	enum fi_cq_format	cq_format;
 	enum fi_wait_obj	comp_wait;  /* must be NONE */
 	uint64_t		remote_cq_data;
+	enum fi_op		op;
+	enum fi_datatype	datatype;
+	size_t			count;
```

Comment:
gurgle that's a lot of additions.  Please move these into an xcontrol_atomic sub-structure, just to keep them grouped.